src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -195,6 +185,64 @@ public function deleteAction($id)
     }
 
     /**
+     * Trigger an action for given media. Action is specified over get-action parameter
+     * @Post("media/{id}")
+     * @param int $id
+     * @param Request $request
+     * @return Response
+     */
+    public function postTriggerAction($id, Request $request)
+    {
+        $action = $this->getRequestParameter($request, 'action', true);
+
+        try {
+            switch ($action) {
+                case 'move':
+                    return $this->moveEntity($id, $request);
+                    break;
+                case 'new-version':
+                    return $this->saveEntity($id, $request);
+                    break;
+                default:
+                    throw new RestException(sprintf('Unrecognized action: "%s"', $action));
+            }
+        } catch (RestException $ex) {
+            $view = $this->view($ex->toArray(), 400);
+
+            return $this->handleView($view);
+        }
+    }
+
+    /**
+     * Move an entity to another collection
+     * @param int $id
+     * @param Request $request
+     * @return Response
+     */
+    protected function moveEntity($id, Request $request)
+    {
+        try {
+            $locale = $this->getLocale($request);
+            $destination = $this->getRequestParameter($request, 'destination', true);
```

Comment:
I think we have said that is shouldnt be abbreviations and in the NodeController i think we used destination!